ArchiSteamFarm/NLog/Targets/SteamTarget.cs

```diff
@@ -59,6 +55,14 @@ internal sealed class SteamTarget : AsyncTaskTarget {
 	// Keeping date in default layout also doesn't make much sense (Steam offers that), so we remove it by default
 	public SteamTarget() => Layout = "${level:uppercase=true}|${logger}|${message}";
 
+	protected override void InitializeTarget() {
+		base.InitializeTarget();
+
+		if (SteamID == 0) {
```

Comment:
Probably worth adding a check if `new SteamID(SteamID)` is `IsIndividualAccount` in case of `ChatGroupID == 0` and `IsChatAccount` otherwise (unless it is actually possible to send messages to other account types).

Also this makes `SteamID` check in `WriteAsyncTask` redundant.

Reply:
`IsChatAccount` doesn't work in this case IIRC, in my group I have `412-356466`, first one is `chatGroupID`, second `chatID`, neither returns `true` for `IsChatAccount`.

The other suggestion is good.